package.json

```diff
@@ -25,12 +25,14 @@
     "generate:nerdctl-stub": "powershell scripts/windows/generate-nerdctl-stub.ps1",
     "build": "node ./scripts/build.mjs",
     "sign": "node scripts/ts-wrapper.js scripts/sign.ts",
-    "test": "npm run lint:nofix && npm run test:unit",
+    "test": "npm run lint:nofix && npm run test:unit && npm run test:extra",
```

Comment:
Before, the `test:unit` depended on the lint which makes sense, but, just curious if we don't want to run `test:extra` if `test:unit` fails? or better way of putting it, does `test:extra` depends on the `test:unit`?

Reply:
No, it currently just does the API spec thing, which is independent of linting.  That is, it may pass even if linting fails.